source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -541,6 +542,14 @@ def __init__(self, output_type_serializer):
             hidden=True,
             help="choose which method is used to filter olympics journeys",
         )
+        parser_get.add_argument(
+            "_olympics_sites_fictitious_fallback[]",
```

Comment:
It's not really a fiction :rofl: I'd go for `virtual_fallback` or `fallback_override` :shrug: